dune

```diff
@@ -0,0 +1,20 @@
+(env
+ (gprof
+  (ocamlopt_flags
+   (:standard -g -p -w -39)
```

Comment:
I think `-g` is there by default, otherwise backtraces wouldn't work, but it doesn't hurt to be explicit.

Reply:
According to the logs, we are still compiling with `-g`.